performance-test/README.md

```diff
@@ -0,0 +1,120 @@
+* Download scripts for the required module.
+* Start JMeter by running the jmeter.bat file for Windows or jmeter file for Unix. 
+* Validate the scripts for one user.
+* Execute a dry run for 10 min.
+* Execute performance run with various loads in order to achieve targeted NFR's.
+
+### Setup points for Execution
+
+* We need some jar files which needs to be added in lib folder of jmeter, PFA dependency links for your reference : 
+
+   * jmeter-plugins-manager-1.10.jar
+      *<!-- https://mvnrepository.com/artifact/kg.apc/jmeter-plugins-manager -->
+<dependency>
+    <groupId>kg.apc</groupId>
+    <artifactId>jmeter-plugins-manager</artifactId>
+    <version>1.10</version>
+</dependency>
+
+   * jmeter-plugins-synthesis-2.2.jar
+      * <!-- https://mvnrepository.com/artifact/kg.apc/jmeter-plugins-synthesis -->
+<dependency>
+    <groupId>kg.apc</groupId>
+    <artifactId>jmeter-plugins-synthesis</artifactId>
+    <version>2.2</version>
+</dependency>
+
+
+* eSignet-default properties: Update the value for the properties according to the execution setup. Perform the execution for eSignet api's with redis setup. So check for the redis setup accordingly. 
+	
+	* Enabling redis cache of eSignet application.
+		*spring.cache.type=redis
+		*spring.redis.host=redis-master-0.redis-headless.redis.svc.cluster.local
+		*spring.redis.port=6379
+		*management.health.redis.enabled=false
+	
+	* Increasing the expire time to 24 hours for field mosip.esignet.cache.expire-in-seconds
+		*'clientdetails' : 86400
+		*'authcodegenerated': 86400
+		*'authtokens': 86400
+		
+* id-authentication-default.properties: Update the value for the properties according to the execution setup. 	
+		*mosip.ida.kyc.token.expire.time.adjustment.seconds=86400
+
+### Execution points for eSignet Authentication API's
+
+*InjiWeb_Test_Script.jmx.jmx
+	
+	* Create Identities in MOSIP Authentication System (Setup) : This thread contains the authorization api's for regproc and idrepo from which the auth token will be generated. There is set of 4 api's generate RID, generate UIN, add identity and add VID. From here we will get the VID which can be further used as individual id. These 4 api's are present in the loop controller where we can define the number of samples for creating identities in which "addIdentitySetup" is used as a variable. 
+	
+	* S01 User accessing Inji Web portal landing page (Execution) :
+		* S01 T01 Issuers : This thread executes issuer endpoint.
+		
+	* S02 User choosing an issuer and landing on credential types screen (Execution):
+		* S02 T01 Issuers : This thread executes issuer endpoint.
+		* S02 T02 Issuer Id : This thread executes eSignet issuer endpoint.
+		* S02 T03 Credential Types : This thread executes credential type endpoint.
+		
+	* S03 User downloads MOSIP National ID credential (Preparation) : This thread generates authcode-token for the download of National ID department PDF.
```

Comment:
the relevace about the samples of preparation and execution is not clear. Can I create 1000 preparation and reuse it ?

Reply:
Updated the Readme file for the procedure of the calculation of number of testdata required.

Reply:
Still confusing " We need to create unique set of testdata for the get-token endpoint for every 24 hours." - Is it valid for 24 hours ?

Reply:
@vishwa-vyom .. The token values will be valid only for 24 hours.

Reply:
The question is can i generate once and use it multiple times within 24 hours ?